Add Pre-release channel and move the Ares (Beta) channel to the last

SadPencil opened the pull request:
<img width="1767" height="1478" alt="图片" src="https://github.com/user-attachments/assets/aedd15c7-df61-4cc5-a407-537b309f65a7" />
<img width="662" height="406" alt="图片" src="https://github.com/user-attachments/assets/33f74c79-1a6e-4a10-9d2f-b6458c73c7b3" />

SadPencil committed:
Add RC channel and move the Ares (Beta) channel to the last

SadPencil committed:
Add 'rc' branch to deployment.yml

SadPencil committed:
Update lblUpdateChannelDescription

SadPencil committed:
Update zh-Hans names for update chanels

SadPencil committed:
Update updateconfig.ini

SadPencil committed:
Update zh-Hans and ru translations for UpdateMirrors

CnCRAZER commented on package/Resources/Config/UpdaterConfig_net48.ini:
Should this file also be added to delete portion of updateexec?

SadPencil replied:
Done via 27374de58ed96b1e9d9bffa687b07bec17f1cd95

SadPencil committed:
Update updateexec

SadPencil committed:
Rename to Beta (Ares)

SadPencil committed:
Update Translation.ini

DeathFishAtEase committed:
[Minor] handled an alignment issue just to reduce the risk of parsing errors

Because I can't guarantee that I'll remember to do this work before it's needed if I don't handle it in advance.

alexp8 commented:
Most people don't know what "RC" is, is there a more user friendly tag for players to recognize?

SadPencil commented:
PreRelease
ReleaseCandidate

SadPencil committed:
RC->Pre-release

SadPencil committed:
Remove obsolete Russian translation comment

SadPencil commented:
@alexp8 Updated to "Pre-release". Your review has been automatically cancelled due to the update :)

Plan to merge this PR after the incoming YR release

alexp8 commented:
cool

brichardson1991 commented on .github/workflows/deployment.yml:
```diff
@@ -7,6 +7,7 @@ on:
   push:
     branches:
       - develop
+      - rc
```
- Pre-Release

SadPencil replied:
resolved

brichardson1991 commented on package/Resources/Config/UpdaterConfig_RC.ini:
Call this file UpdaterConfig_PreRelease

SadPencil replied:
resolved

brichardson1991 commented on package/Resources/OptionsWindow.ini:
```diff
@@ -23,21 +23,22 @@ Size=133,23
 RestartRequired=true
 Item0File0=Resources/Config/UpdaterConfig_Live.ini,updateconfig.ini,OverwriteOnMismatch
-Item1File0=Resources/Config/UpdaterConfig_Beta.ini,updateconfig.ini,OverwriteOnMismatch
+Item1File0=Resources/Config/UpdaterConfig_RC.ini,updateconfig.ini,OverwriteOnMismatch
```
UpdaterConfig_PreRelease

SadPencil replied:
resolved

brichardson1991 reviewed (commented):
If we're going to call it pre-release the files should reflect that, else we're going to cause confusion

CnCRAZER commented on package/updateexec:
```diff
@@ -515,6 +515,8 @@ Maps\Yuri's Revenge\blitz_park_of_vytautas_blitz.png
 INI\ForcedOptions ReadMe.txt
 Maps\Custom\83b65056362a027b56ce57842588d1e470373af3.png
+Resources/Config/UpdaterConfig_net48.ini
+Resources/Config/UpdaterConfig_Beta.ini
```
`Resources/Config/UpdaterConfig_Beta.ini` should be `Resources\Config\UpdaterConfig_Beta.ini`? Not sure if it matters. Best to be consistent IMO

SadPencil replied:
resolved

brichardson1991 reviewed (changes requested):
If we're going to call it pre-release the files should reflect that, else we're going to cause confusion

SadPencil committed:
Update updateexec

SadPencil committed:
Rename to UpdaterConfig_PreRelease.ini

SadPencil committed:
Update branch name

SadPencil committed:
Merge branch 'develop' into add-rc

brichardson1991 commented:
Thanks @SadPencil

SadPencil committed:
Forgot to update the url
